src/cf-typography/src/cf-typography.less

```diff
@@ -11,7 +11,8 @@
 @webfont-regular: 'AvenirNextLTW01-Regular';
 @webfont-italic: @webfont-regular;
 @webfont-medium: 'AvenirNextLTW01-Medium';
-@webfont-demi: 'AvenirNextLTW01-Demi';
+@webfont-demi: @webfont-regular;
+// @webfont-demi: 'AvenirNextLTW01-Demi';
```

Comment:
Can this be deleted?

Reply:
Yep, that's why this is still a WIP :) I'll ping you when it's ready for review!